Kickstarter-iOS/ViewModels/ChangePasswordViewModelTests.swift

```diff
@@ -151,7 +151,7 @@ final class ChangePasswordViewModelTests: TestCase {
 
       scheduler.advance()
 
-      self.changePasswordFailureObserver.assertValues(["Error changing password"])
+      self.changePasswordFailureObserver.assertValueCount(1)
```

Comment:
made this change because the test was not passing on my branch but on master it was. would appreciate feedback on this. thank you!